### PR TITLE
JBIDE-20884 : OpenShift New and Noteworthy for 4.3.0.Final

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+it will add an option to create OpenShift 3 connections in the OpenShift Connection wizard.
+
+image::./images/oseval_sign_in_to_server.png[]
+
+http://www.openshift.org/#v3["OpenShift 3"] is not available as an online offering yet. Please refer to the link:/documentation/howto/openshift3_getting_started.html[Getting Started] documentation
+
+OpenShift 3 tooling supports 2 authentication schemes:
+
+- Basic Authentication: type in you login/password
```
Comment (mishaone) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Final.adoc]: typo: s/you/your

File: documentation/whatsnew/openshift/openshift-news-3.0.0.Final.adoc [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+image::./images/manage-projects-link.png[]
+
+If an application is created without a project, the tools prompt you to create a project for the application beforehand:
+
+image::./images/manage-projects-wizard.png[]
+
+related_jira::JBIDE-20016[]
+
+=== Manually Trigger Builds
+Automatic build trigger require using Webhook triggers. However, you can still manually trigger builds when selecting your Build Configs in the OpenShift Explorer.
```
Comment (mishaone) [inline documentation/whatsnew/openshift/openshift-news-3.0.0.Final.adoc]: s/Automatic build trigger/Automatic build triggers

**Comment by mishaone** [comment]
Two minor typos, otherwise looks good, @fbricon!

**Comment by alexeykazakov** [comment]
Fixed the typos and pushed to master and production.

**Pull request closed without merging**